.travis.yml

```diff
@@ -30,7 +30,6 @@ before_script:
 
 script:
   - make install
-  - make lint
   - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
         dbus-launch pytest;
```

Comment:
can we have a comment here? so that we don't go over that conversation again?

Reply:
Sure.